cider/util/Regexp.cpp

```diff
@@ -43,7 +44,7 @@ extern "C" RUNTIME_EXPORT bool regexp_like(const char* str,
     boost::regex re(pattern, pat_len, boost::regex::extended);
     boost::cmatch what;
     result = boost::regex_match(str, str + str_len, what, re);
-  } catch (std::runtime_error& error) {
+  } catch (std::exception& error) {
```

Comment:
why change this.

Reply:
CiderException inherits from std::exception.

Reply:
better keep it. https://en.cppreference.com/w/cpp/error/exception

Reply:
fix